frontend/src/pages/Subscribers/SubscriberOverview.js

```diff
@@ -46,7 +48,7 @@ class SubscriberOverview extends Component {
         })
       });
     }
-    delete origiData.FlowRules;
+    // delete origiData.FlowRules;
```

Comment:
Why remove this line?

Reply:
Since the current Qosflows doesn't correctly add the flow rules, I needed the flow rules that were saved in the origidata